content/post/glue-work.md

```diff
@@ -0,0 +1,79 @@
+
+These disconnects represent a misunderstanding between the company as a whole and the analytics team about what is valuable, and what is valued. Clearly identifying and communicating why different types of analytical work is valued within the team sets the stage to more clearly communicate the value of the analytics team itself to the rest of the company.
+
+## So What Do We Do About It?
+The problem of glue work is fixable. If your team is doing work that helps improve the likelihood of success on projects, that work needs to be explicitly valued.
+
+### Managers
+If you are in a position to drive systematic change in your organization, you need to:
+- Figure out what your team is doing that is not written in their job descriptions. Watch them, and ask them - about themselves and their teammates. Separate the list into work that requires an analyst’s skills and improves the output of the analytics team, and work that doesn’t.
+- Work that is truly valuable to your team’s mandate? Write clear career ladders that reward what needs to be done - including glue work. This might include documentation, relationship-building, onboarding new team members, and building subject matter expertise. I have a lot more to say on this - enough to fill another post, coming soon.
+- Work that needs to be done, but is not particularly relevant to your team’s mandate? There will always be tasks that are non-promotable but need to get done, like scheduling meetings or planning team off-sites. Either you do that work as team manager, or explicitly assign a fair rotation of these tasks within your team. Don’t wait for volunteers.
+- Understand where your team members want to go. Are they trying to get from analyst to senior analyst? Do they want to get from analyst to engineer? Or do they ultimately want to move from analytics into product or marketing? Not everyone wants to climb the same ladder, and sometimes work that is not explicitly promotable in analytics is still helpful in getting that person where they want to be. Just make sure those choices are clear to your team members as you work with them on development plans.
+- If there is enough of it, what used to be glue work can become its own role. That can be a rotating role, like a daily [triager](https://about.gitlab.com/handbook/business-ops/data-team/#triager) who answers questions, acknowledges inbound requests, and begins the process of estimating and prioritizing. In can be a whole new analytics role, like an [analytics engineer](https://www.locallyoptimistic.com/post/analytics-engineer/) who fills the gaps between analysts and data engineers. Or for coordination, scheduling, and team management, maybe you need a project manager or a team coordinator for whom excelling at these tasks is their core focus and helps them achieve their career goals.
```

Comment:
Data Product Manager is a thing many teams have for this specific purpose

Reply:
Good call.